lib/rescue_from_unless_local.rb

```diff
@@ -35,7 +37,12 @@ def rescue_from_unless_local(*klasses, with: nil, &block)
             end
           end
 
-        raise exception if Rails.application.config.consider_all_requests_local
+        if Rails.application.config.consider_all_requests_local
```

Comment:
we should prob. have a unit test for this file that makes sure the right messages are sent out (to Raven) if sentry is turned on.

Reply:
agreed, but deferring to moving this into a gem (https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/673)